gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var gulp     = require('gulp'),
+    mocha    = require('gulp-mocha'),
+    babel    = require('babel'),
+    plumber  = require('gulp-plumber'),
+    istanbul = require('gulp-istanbul');
```

Comment:
The logical grouping can be separated by a space. It's similar issue as trailing commas. I want to be able to add something anywhere in the list, and I can't without special rules. If we start every line with `var`, and end it with a `;`, it's uniform throughout.